l10n_ar_currency_update/__manifest__.py

```diff
@@ -19,7 +19,7 @@
 ##############################################################################
 {
     'name': 'Argentinian Currency Rate Update',
-    'version': "15.0.1.0.0",
+    'version': "15.0.1.1.0",
```

Comment:
In this case, we are not updating the database ( model, views or data). So we do not need to increase the module version 
```suggestion
    'version': "15.0.1.0.0",
```